e2e/cypress/support/ui_commands.js

```diff
@@ -134,6 +134,7 @@ Cypress.Commands.add('postMessageReplyInRHS', (message) => {
 });
 
 function waitUntilPermanentPost() {
+    cy.get('#postListContent').should('be.visible');
```

Comment:
This verification ensure that the channel view has post list content loaded instead of another case which is loading (`Loading...`)